templates/base.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
```

Comment:
Please be mindful of the formatting here. HTML tags should start and end with the same indentation and they shouldn't intersect with Python code blocks. See comments below.